tutorials/chartjs/label-color.md

```diff
@@ -0,0 +1,119 @@
+With chartjs version 3.7.1, you can change the color of the labels.
+In version 3, chartjs changed how to configure the x and y axis of the graph.
+Instead of a array, it is now an object.
+Do not use version 2 as there is a bug where the x-axis options point to the y-axis options and vice versa.
+
+```html
+<style>
+#chart-wrapper {
+    display: inline-block; position: relative; height: 400px; width: 66%;
+}
+
+</style>
+
+
+<div id="chart-wrapper">
+  <canvas id="chart"></canvas>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.7.1/chart.min.js"></script>
+
+<script>
+
+  const ctx = document.getElementById('chart').getContext('2d');
+
+  const chart = new Chart(ctx, {
+      // The type of chart we want to create
+      type: 'bar',
+       data: {
+        labels: ["Red", "Blue", "Yellow", "Green", "Purple", "Orange"],
+        datasets: [{
+            label: 'Example Data',
+            data: [12, 19, 3, 5, 2, 3],
+            backgroundColor: [
```

Comment:
Let's get rid of the `backgroundColor` `borderColor`, and `borderWidth` options. In this tutorial, we're concerned with the axis label colors, not the actual chart colors.